shell/platform/darwin/ios/framework/Source/FlutterViewTest.mm

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterView.h"
+
+@interface FakeDelegate : FlutterEngine
+@property(nonatomic) BOOL ensureSemanticsEnabledCalled;
+@end
+
+@implementation FakeDelegate
+
+- (void)ensureSemanticsEnabled {
```

Comment:
This method has to be updated, this test should be failing.

Reply:
No I inherited from the FlutterEngine so I only need to override ensureSemanticsEnabled. I want to test the FlutterView.isAccessibilityElement call eventually lead to ensureSemanticsEnabled

Reply:
otheriwse, I will need to write two tests.
one for FlutterView.isAccessibilityElement calls  FlutterViewEngineDelegate.flutterViewAccessibilityDidCall

the other one for  FlutterEngine.flutterViewAccessibilityDidCall calls FlutterEngine.ensureSemanticsEnabled

Reply:
Ahh yea, sorry missed that.  I think you should use a mock object to keep the test hermetic.  Now you're tying your testing of functionality on FlutterView to implementation details in FlutterEngine which is fragile.  It easy to replace it if you use OCMock, there are examples in other tests.